src/learning/cyclegan.py

```diff
@@ -0,0 +1,96 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+from modules import CycleGANLoss, Discriminator, Generator
+import utils
+
+
+def train():
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    torch.manual_seed(7)
+    n_residual_blocks = 9
+    use_dropout = False
```

Comment:
We'll probably want to use ArgumentParser, to pass these parameters as command line arguments. Can be done like this: https://github.com/pytorch/examples/blob/master/snli/util.py

Reply:
Most definitely. My thoughts were that we should postpone this to a point in time in which we are at least somewhat confident we can estimate what parameters might be of interest to be adapted.
I figured this point in time was not now. Do you disagree?